moped-editor/src/components/LookupSelectComponent.js

```diff
@@ -24,10 +24,22 @@ const LookupSelectComponent = (props) => {
   return (
     <Select
       variant="standard"
-      style={{ minWidth: "8em" }}
       id={props.name}
       value={value || props.defaultValue}
       ref={ref}
+      disableUnderline
+      sx={{
+        minWidth: "8em",
+        padding: "10px",
```

Comment:
I'm cool with bumping this down to 8px so you could use `theme.spacing(1)`. I think it would be cool to move towards using the theme values when possible but don't want to fixate on it too much!